aries-site/src/examples/templates/dashboards/content/RecentActivity.js

```diff
@@ -70,7 +70,7 @@ export const RecentActivity = () => {
         (previous, current) => {
           const { title, createdAt, selfUri } = current;
           const next = { ...previous };
-          const count = previous[title]?.count + 1 || 1;
+          const count = previous[title]?.count ? previous[title].count + 1 : 1;
```

Comment:
What is this rule? Not sure if I like this one or not.

Reply:
This rules blocks that the result could be NaN (undefined + 1 maybe?). need to investigate more.

Reply:
Was wondering if that what it was. If so, better to keep. I believe we kept it in Grommet.

Reply:
Yes we did, I matched Grommet rules.

Reply:
I cant remember if we disabled this rule in grommet or not but it was a strange one

Reply:
oh you answered my question I wasnt sure if we kept it